_posts/2019-04-26-workshop-rails-upgrade.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Workshop - Upgrade Rails 101: The Roadmap to Smooth Upgrades"
+date: 2019-04-05 10:00:00
+categories: ["upgrade-rails", "workshop"]
+author: "luciano"
+---
+
+Next week the [Rails](https://rubyonrails.org/) community is having the biggest event of the year, the [Rails Conf 2019](https://railsconf.com/). We participated on previous editions of this conference but this time we have the honor to be in the speakers side of it. We'll be teaching you what we're best at: [Rails Upgrades](https://fastruby.io/blog/tags/upgrades).
```

Comment:
"We participated IN previous editions of this conference but this time we have the honor to be ON the speakers side of it."